Arrowgene.Ddon.GameServer/Handler/QuestCancelHandler.cs

```diff
@@ -15,26 +15,39 @@ public QuestCancelHandler(DdonGameServer server) : base(server)
         }
 
         public override S2CQuestQuestCancelRes Handle(GameClient client, C2SQuestQuestCancelReq packet)
-        {                
-            var quest = client.Party.QuestState.GetQuest(packet.QuestScheduleId);
+        {
+            var quest = QuestManager.GetQuestByScheduleId(packet.QuestScheduleId);
+            var questState = quest.IsPersonal ? client.QuestState : client.Party.QuestState;
```

Comment:
Should we make a helper function for this kind of logic? Seems it isn't the first or last time we would need to do this? Maybe a static function to the `QuestManager.GetQuestState(client, quest)`?